release/ray_release/config.py

```diff
@@ -47,10 +52,25 @@ def read_and_validate_release_test_collection(
 ) -> List[Test]:
     """Read and validate test collection from config file"""
     with open(config_file, "rt") as fp:
-        test_config = yaml.safe_load(fp)
+        tests = parse_test_definition(yaml.safe_load(fp))
 
-    validate_release_test_collection(test_config, schema_file=schema_file)
-    return test_config
+    validate_release_test_collection(tests, schema_file=schema_file)
+    return tests
+
+
+def parse_test_definition(test_definitions: List[TestDefinition]) -> List[Test]:
+    tests = []
+    for test_definition in test_definitions:
+        if "flavors" not in test_definition:
+            tests.append(test_definition)
+            continue
+        flavors = test_definition.pop("flavors")
```

Comment:
report error when `flavors` is an empty list?

Reply:
Good catch!